cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -438,12 +438,20 @@ func (s *Server) vacuum() {
 				debug.Printf("failed getting all file paths for %s", path)
 				continue
 			}
+			var shards []shard
+			if repos, _, err := zoekt.ReadMetadataPathAlive(path); err == nil {
+				shards = make([]shard, 0, len(repos))
```

Comment:
minor: don't really need to pre-allocate, this will usually be small and short lived?